os/kernel/binary_manager/Kconfig

```diff
@@ -3,6 +3,12 @@
 # see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
 #
 
+config USE_BP
```

Comment:
May be we can name it as
BOOTLOADER_FLAT
BOOTLOADER_V4 (based on BL_VER)
BOOTLOADER_V3 (based on 3.1/4.0)
or something more appropriate ?

Reply:
Thanks for the suggestions. Earlier the config name was CONFIG_NEW_BL. I have changed it to CONFIG_USE_BP. I think instead of bootloader versions, we can differentiate them better with and without the use of boot parameters. Please let me know your thoughts.